DependencyInjection/Configuration.php

```diff
@@ -45,8 +45,7 @@ public function getConfigTreeBuilder()
                             ->defaultValue(array('Default'))
                         ->end()
                         ->scalarNode('subject_provider')->defaultValue('mremi_contact.subject_provider.noop')->cannotBeEmpty()->end()
-                        ->booleanNode('captcha_disabled')->defaultFalse()->end()
-                        ->enumNode('captcha_type')->defaultValue('genemu_captcha')->values(array('genemu_captcha', 'genemu_recaptcha'))->end()
+                        ->scalarNode('captcha_type')->defaultFalse()->end()
```

Comment:
I think it's a regular practice to have string|false. But how to disable it in dev than? Redefine with empty string?

User may not want to use captcha at all. Why to force him?

>By the way, what tests for configuration.php are testing? There is already a constraints like >cannotBeEmpty or isRequired.

I think this particular test is useless, all they testing is how already tested config component returning default values.  Is there purpose for it?